app/lib/frontend/search_service.dart

```diff
@@ -210,12 +218,16 @@ class SearchQuery {
     this.offset: 0,
     this.limit: 10,
     this.type,
+    this.packagePrefix,
     this.bias,
   });
 
   /// Returns the query text to use in CSE.
   String buildCseQueryText() {
     String queryText = text;
+    if (packagePrefix != null) {
+      queryText += ' $packagePrefix';
```

Comment:
So if the prefix is not explicit, wouldn't this mean the search term is duplicated here?